vpn-proxy/app/tunnels.py

```diff
@@ -292,4 +292,4 @@ def stop_tunnel(tunnel):
     del_rtable(tunnel.id, tunnel.rtable)
     stop_openvpn(tunnel.name)
     remove_file(tunnel.conf_path, 'conf file')
-    remove_file(tunnel.key_path, 'key file')
+    remove_file(tunnel.key_path, 'key file')
```

Comment:
There should be a line break (not empty line) in every file, configure your editor accordingly.